pages/manager_page.py

```diff
@@ -0,0 +1,70 @@
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.ui import WebDriverWait
+
+from pages.locators import AllLocators
+
+
+class ManagerPage:
+    def __init__(self, browser):
+        self.browser = browser
+        self.url = "https://www.globalsqa.com/angularJs-protractor/BankingProject/#/manager"
```

Comment:
Здесь лучше разделить логику на два или более класса.
```suggestion
class BasePage:
    def __init__(self, browser, url):
        self.browser = browser
        self.url = url
```